content/developer/reference/backend/views.rst

```diff
@@ -1931,6 +1933,25 @@ Possible children elements of the list view are:
 
   A special button (`type="edit"`) can be defined to open the many2one form view.
 
+``header``
+  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.
+
+  .. code-block:: xml
+
+      <header>
+          <button name=toDo" type="object" string="Always Display" display="always"/>
+          <button name="toDo" type="object" string="Selection Display"/>
+      </header>
+
+  ``button``
+    as :ref:`button <reference/views/list/button>` with:
+
+    ``display``
+      By default, those buttons are only displayed when some records are
+      selected, and they apply on the selection. When the attribute ``display``
+      is set to ``always``, the button is available all the time, even if there's
+      no selection.
```

Comment:
RST is very sensitive to indentation. Even 1 extra space creates a description list in HTML, which is probably not intended here.

Reply:
We discussed it with aab and we prefer our solution. We find it more clear.

Reply:
> RST is very sensitive to indentation. Even 1 extra space creates a description list in HTML, which is probably not intended here.

Are you sure for this fix ?

Reply:
I'm not sure to understand the question. Am I sure that this is the RST syntax for a description list? Then yes^^

What are you trying to achieve here with that extra space? I can't tell if it's a mistake or an attempt at reproducing a structure you saw somewhere else?

Reply:
I do the same as in the rest of the code. I'm not sure I understand.
```
``header``
  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.
  When used within a header, buttons accept the extra attribute :ref:`display <...>`.
```

Reply:
Ok, I see! Then you must insert `.. rst-class:: o-definition-list` before the code block on line 1939 and before \``display`` on line 1949. This applies the appropriate styling on description lists, which are used as a hack to list attributes.